src/components/Authentication/LoginPage.scss

```diff
@@ -0,0 +1,127 @@
+.logo {
+    padding-top: 1%;
+    padding-left: 1.5%;
+    .title {
+        position: absolute;
+        font-family: "Roboto Slab", serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 24px;
+        line-height: 26px;
+        text-decoration: none;
+        color: white;
+    }
+}
+.logo2 {
+    padding-top: 1%;
+    padding-left: 1.5%;
+    .title {
```

Comment:
If this `.title` class is the same as the one on line 4, just take take out of `.logo2` and `.logo` individually and do this:
```
.logo, .logo2 {
    .title { //title styling }
}
```